File: android-java/RetrofitJavaJson/app/src/main/java/com/tnqr/retrofitjavajson/service/CryptoAPI.java
package com.tnqr.retrofitjavajson.service;

import com.tnqr.retrofitjavajson.model.CryptoModel;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface CryptoAPI {
    //GET //POST //UPDATE //DELETE
    //base url = www.website.com
    //

    @GET("atilsamancioglu/K21-JSONDataSet/blob/master/crypto.json")
     Call<List<CryptoModel>> getData();

    //


}

File: android-kotlin/CoroutineAndRetrofit/app/src/main/java/com/example/coroutineandretrofit/service/CryptoAPI.kt
package com.example.coroutineandretrofit.service

import com.example.coroutineandretrofit.model.CryptoModel
import retrofit2.Call
import retrofit2.Response
import retrofit2.http.GET

interface CryptoAPI {
    @GET("atilsamancioglu/K21-JSONDataSet/master/crypto.json")
    suspend fun getData(): Response<List<CryptoModel>>

}